source/Activated.js

```diff
@@ -1,6 +1,6 @@
-import React, { useCallback, useEffect, useRef, useState } from 'react'
 import { bool, func, node, object, string } from 'prop-types'
-import { useWindowSize } from './helpers'
+import { useWindowSize } from 'react-use'
```

Comment:
Per the [Usage docs](https://github.com/streamich/react-use/blob/master/docs/Usage.md), maybe should do it like
```js
import useWindowSize from 'react-use/lib/useWindowSize'
```
so we only pull that thing in?